modules/installation-vsphere-installer-infra-requirements.adoc

```diff
@@ -3,11 +3,11 @@
 // * installing/installing_vsphere/ipi/ipi-vsphere-installation-reqs.adoc
 // * installing/installing_vsphere/upi/upi-vsphere-installation-reqs.adoc
 
```

Comment:
I think it is missing the mod docs content type field, ID, and module title: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata

Reply:
Sorry, it's not missing the ID - I see it below, but the title needs to be a Level 1 heading.